src/main/java/com/sap/sgs/phosphor/fosstars/model/score/oss/LgtmScore.java

```diff
@@ -31,14 +36,14 @@
     GRADE_TO_POINTS.put(LgtmGrade.B, 6.0);
```

Comment:
Any reason for A_PLUS and A having the same Point?

Reply:
I think they both are good and deserve the same number of points.